app/build.gradle.kts

```
@@ -0,0 +1,39 @@
+@file:Suppress("DSL_SCOPE_VIOLATION", "UnstableApiUsage")
+
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.dagger.hilt)
+    alias(libs.plugins.kotlinx.serialization)
+    alias(libs.plugins.kotlin.kapt)
+}
+
+android {
+    namespace = "com.yapp.all_rounder_3"
```

```suggestion
    namespace = "com.yapp.allrounder3"
```
코틀린컨벤션에서는 언더바를 안써서, 제거하는게 어떨까?

---

고마워! 언더바 제거하고 패키지 명이랑 맞춰놓을게